server/sandbox.go

```diff
@@ -172,6 +172,7 @@ func (s *Server) RunPodSandbox(ctx context.Context, req *pb.RunPodSandboxRequest
 		if err != nil {
 			return nil, err
 		}
+		g.SetProcessSelinuxLabel(processLabel)
 	}
```

Comment:
@rhatdan we regenerate that when we load the container's sandbox mountLabel from the processLabel so need to save that